Assets/Scripts/Throw/NextColor.cs

```diff
@@ -11,16 +14,22 @@ public class NextColor
     public Queue<ColorSO> NextColorQueue = new();
     public Queue<Color> NextColors = new();
     public Queue<GameObject> NextYarns = new();
-
+    int colorindex = 0;
+    bool _e;
```

Comment:
I don't know what `_e` is doing, is this a variable being used somewhere? May want a more descriptive name, since this doesn't tell me much about what this is used for.